src/analytics.js

```diff
@@ -0,0 +1,92 @@
+if (typeof analytics !== 'undefined') {
+    analytics.page('Landing', {
+        variation: PAGE_NAME
+    })
+}
+
+for (const delay of [5, 15, 30, 45]) {
+    setTimeout(() => sendEvent('TimeOnPageThreshold Reached', {
+        page: PAGE_NAME,
+        threshold: delay.toString()
+    }), delay * 1000)
+}
+
+
+function getBlockProperties(block) {
+    const blockVariants = {
+        FAQs: 'vertical-expanding',
+        footer: 'basic',
+        inspirations: 'carousel',
+        interviewers: 'carousel',
+        'packages-and-pricing': 'basic',
+        testimonials: 'carousel',
+        ticker: 'basic',
+        topnav: 'basic',
+        'whats-included': 'basic',
+        'your-child': 'basic'
+    }
+    const additionalProperties = {}
+    if (block === 'topnav') {
+        additionalProperties['topnav-pinned'] = (window.pageYOffset || document.body.scrollTop) > 0
+    }
+    return { 'block-variant': blockVariants[block], ...additionalProperties }
+}
+
+
+function getFAQEventProperties(target) {
+    const questionText = $(target).children().first().children().first().text()
+    return { block: 'FAQs', name: questionText, type: 'question' }
+}
+
+
+function getInterviewerPlayerEventProperties(target) {
+    const interviewerCard = $(target).closest('.interviewers-block')
+    const interviewerName = $($(interviewerCard).find('.heading-6')[0]).text()
+    return {
+        content_type: 'interviewer_audio',
+        name: interviewerName
+    }
+}
+
+
+// Take the target of an event and return an object of the relevant properties to be included in the tracking event.
+function getEventProperties(eventName, dataset, target) {
+    const eventProperties = {}
+
+    // Add custom attributes that start with `data-event-` to the event properties.
+    for (const attr in dataset) {
+        if (attr.startsWith('event') && attr !== 'eventName') {
```

Comment:
The comment mentions that the attribute should start with `data-event`, but here it seems that the condition is `event`

Reply:
The element's dataset automatically parses its custom attributes into more concise names. So the `data-event-name` HTML attribute is accessed with `eventName` in the dataset.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset

Reply:
Ha good to know!